### PR TITLE
VDB-2021 - 2.17 release

### PR DESCRIPTION
**Pull request by elizabethengelman**

### CODE REVIEW

File: environments/testing.toml [hunk 1]
```diff
@@ -487,38 +487,38 @@
...:"tick","outputs":[],"payable":false,"stateMutability":"nonpayable","type":"function"},{"inputs":[{"internalType":"address","name":"vat_","type":"address"},{"internalType":"bytes32","name":"ilk_","type":"bytes32"}],"payable":false,"stateMutability":"nonpayable","type":"constructor"},{"anonymous":false,"inputs":[{"indexed":false,"internalType":"uint256","name":"id","type":"uint256"},{"indexed":false,"internalType":"uint256","name":"lot","type":"uint256"},{"indexed":false,"internalType":"uint256","name":"bid","type":"uint256"},{"indexed":false,"internalType":"uint256","name":"tab","type":"uint256"},{"indexed":true,"internalType":"address","name":"usr","type":"address"},{"indexed":true,"internalType":"address","name":"gal","type":"address"}],"name":"Kick","type":"event"},{"anonymous":true,"inputs":[{"indexed":true,"internalType":"bytes4","name":"sig","type":"bytes4"},{"indexed":true,"internalType":"address","name":"usr","type":"address"},{"indexed":true,"internalType":"bytes32","name":"arg1","type":"bytes32"},{"indexed":true,"internalType":"bytes32","name":"arg2","type":"bytes32"},{"indexed":false,"internalType":"bytes","name":"data","type":"bytes"}],"name":"LogNote","type":"event"}]'
```
Comment (rmulhol) [inline environments/testing.toml]: Any reason not to just delete the config for contracts that aren't in this release?

File: transformers/events/flip_kick/converter_test.go [hunk 2]
```diff
@@ -37,7 +35,6 @@ var _ = Describe("FlipKick Converter", func() {
 
 		Expect(err).NotTo(HaveOccurred())
 
-		models[0].ForeignKeyValues[constants.AddressFK] = strings.ToLower(models[0].ForeignKeyValues[constants.AddressFK])
```
Comment (rmulhol) [inline transformers/events/flip_kick/converter_test.go]: 🥇✂️

File: transformers/integration_tests/flap_kick.go [hunk 3]
```diff
@@ -31,8 +31,7 @@ import (
 	. "github.com/onsi/gomega"
 )
 
-var _ = XDescribe("FlapKick Transformer", func() {
-	//TODO: There are currently no FlapKick events on Kovan
```
Comment (rmulhol) [inline transformers/integration_tests/flap_kick.go]: 🎉

File: transformers/integration_tests/plugin.go [hunk 4]
```diff
@@ -233,7 +234,7 @@ var _ = Describe("Plugin test", func() {
 					var flip string
 					_ = db.Get(&flip, `SELECT flip FROM maker.cat_file_flip WHERE header_id = $1`, headerID)
 					return flip
-				}).Should(Equal(ethFlipAddress))
+				}).Should(Equal(test_data.EthFlipAddress()))
```
Comment (rmulhol) [inline transformers/integration_tests/plugin.go]: 👍

File: transformers/test_data/config_values.go [hunk 5]
```diff
@@ -1,21 +1,35 @@
...
+	return addressesResult
+}
+
+func EthFlipAddress() string    { return checksum(constants.GetContractAddress("MCD_FLIP_ETH_A")) }
+func FlopAddress() string       { return checksum(constants.GetContractAddress("MCD_FLOP")) }
+func JugAddress() string        { return checksum(constants.GetContractAddress("MCD_JUG")) }
+func SpotAddress() string       { return checksum(constants.GetContractAddress("MCD_SPOT")) }
+func VatAddress() string        { return checksum(constants.GetContractAddress("MCD_VAT")) }
+func VowAddress() string        { return checksum(constants.GetContractAddress("MCD_VOW")) }
+func CdpManagerAddress() string { return checksum(constants.GetContractAddress("CDP_MANAGER")) }
```
Comment (rmulhol) [inline transformers/test_data/config_values.go]: Are all of these used? Wondering if we might be able to delete some of them

Comment (m0ar) [reply transformers/test_data/config_values.go]: We should let them stay until we figure out how to nuke them to hell :( It's so backwards to have to construct functions for things that are already in your config

File: environments/testing.toml (hunk 1 above)
Comment (elizabethengelman) [reply environments/testing.toml]: My thought was to keep them around as documentation, but I don't feel super strongly about that, so I'd be happy to remove them!

**Comment by elizabethengelman** [comment]
@m0ar yep! got the ABIs from changelog.makerdao.com

File: environments/testing.toml (hunk 1 above)
Comment (elizabethengelman) [reply environments/testing.toml]: Actually, on second thought, I think it makes sense to remove them. We can always look at the 2.15 release or the [flip initializers](https://github.com/makerdao/vdb-mcd-transformers/tree/staging/transformers/storage/flip/initializers) to see what other flip contracts we've implemented. Removed in 
c5f2f43

**Pull request merged**